debezium-core/src/test/java/io/debezium/relational/history/JdbcDatabaseHistoryTest.java

```diff
@@ -0,0 +1,127 @@
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational.history;
+
+import static io.debezium.relational.history.JdbcDatabaseHistory.JDBC_PASSWORD;
+import static io.debezium.relational.history.JdbcDatabaseHistory.JDBC_URI;
+import static io.debezium.relational.history.JdbcDatabaseHistory.JDBC_USER;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Types;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.debezium.config.Configuration;
+import io.debezium.relational.Column;
+import io.debezium.relational.Table;
+import io.debezium.relational.TableId;
+import io.debezium.relational.Tables;
+import io.debezium.relational.ddl.DdlParser;
+import io.debezium.util.Collect;
+
+/**
+ * @author Ismail simsek
+ */
+public class JdbcDatabaseHistoryTest {
+
+    protected DatabaseHistory history;
+    String dbFile = "/tmp/test.db";
+    String databaseName = "db";
+    String schemaName = "myschema";
+    String ddl = "CREATE TABLE foo ( first VARCHAR(22) NOT NULL );";
+    Map<String, Object> source = Collect.linkMapOf("server", "abc");
+    Map<String, Object> position = Collect.linkMapOf("file", "x.log", "positionInt", 100, "positionLong", Long.MAX_VALUE, "entry", 1);
+    TableId tableId = new TableId(databaseName, schemaName, "foo");
+    Table table = Table.editor()
+            .tableId(tableId)
+            .addColumn(Column.editor()
+                    .name("first")
+                    .jdbcType(Types.VARCHAR)
+                    .type("VARCHAR")
+                    .length(22)
+                    .optional(false)
+                    .create())
+            .setPrimaryKeyNames("first")
+            .create();
+    TableChanges tableChanges = new TableChanges().create(table);
+    HistoryRecord historyRecord = new HistoryRecord(source, position, databaseName, schemaName, ddl, tableChanges);
+    Map<String, Object> position2 = Collect.linkMapOf("file", "x.log", "positionInt", 100, "positionLong", Long.MAX_VALUE, "entry", 2);
+    TableId tableId2 = new TableId(databaseName, schemaName, "bar");
+    Table table2 = Table.editor()
+            .tableId(tableId2)
+            .addColumn(Column.editor()
+                    .name("first")
+                    .jdbcType(Types.VARCHAR)
+                    .type("VARCHAR")
+                    .length(22)
+                    .optional(false)
+                    .create())
+            .setPrimaryKeyNames("first")
+            .create();
+    TableChanges tableChanges2 = new TableChanges().create(table2);
+    HistoryRecord historyRecord2 = new HistoryRecord(source, position, databaseName, schemaName, ddl, tableChanges2);
+    DdlParser ddlParser = new TestAntlrDdlParser();
```

Comment:
This seems a bit uncommon to initialize such large number of fields via direct field assignment in a test. Perhaps move it into a `@BeforeClass` method?